### PR TITLE
Update install guide to mention installCRDs option

### PR DESCRIPTION
**Pull request by munnerz**
Signed-off-by: James Munnelly <james@munnelly.eu>

/assign @meyskens 
/milestone v0.15

### CODE REVIEW

File: content/en/docs/installation/kubernetes.md [hunk 1]
```diff
@@ -131,26 +118,59 @@ Add the Jetstack Helm repository.
...
 
 # Helm v2
 $ helm install \
   --name cert-manager \
   --namespace cert-manager \
   --version v0.15.0 \
-  jetstack/cert-manager
-
+  jetstack/cert-manager \
+  # --set installCRDs=true
```
Comment (meyskens) [inline content/en/docs/installation/kubernetes.md:173]: Did we test this with Helm 2? I do remember we specifically targeted Helm 3 with this.

Comment (munnerz) [reply content/en/docs/installation/kubernetes.md:173]: We're pretty much falling back on Helm's default behaviour by including the CRDs as we do - we don't use the `crd-install` annotation, and the way that Helm handles CRDs in the `templates/` directory has not changed between 2 and 3 :).

Comment (meyskens) [reply content/en/docs/installation/kubernetes.md:173]: okay!

**Pull request merged**